docs/elasticsearch/reindex-data.md

```diff
@@ -0,0 +1,200 @@
+
+To use the `_reindex` operation to change your index settings, create a destination index with your desired settings and then run the above command.
+
+## Reindex from a remote cluster
+
+You can copy documents from an index in a remote cluster. Use the `remote` option to specify the remote hostname and the required login credentials.
+
+This command reaches out to a remote cluster, logs in with the username and password, and copies all the documents from the source index in that remote cluster to the destination index in your local cluster:
+
+```json
+POST _reindex
+{
+   "source":{
+      "remote":{
+         "host":"https://<REST_endpoint_of_remote_cluster>:9200",
+         "username":"YOUR_USERNAME",
+         "password":"YOUR_PASSWORD"
+      }
+   },
+   "dest":{
+      "index":"destination"
+   }
+}
+```
+
+## Reindex a subset of documents
+
+You can copy only a specific set of documents that match a search query.
+
+This command copies only a subset of documents matched by a query operation to the destination index:
+
+```json
+POST _reindex
+{
+   "source":{
+      "index":"source",
+      "query":"Add your query operation"
+   },
+   "dest":{
+      "index":"destination"
+   }
+}
+```
+
+For a list of all query operations, see [Full-text queries](../full-text/).
+
+## Combine one or more indices
+
+You can combine documents from one or more indices by adding the source indices as a list.
+
+This command copies all documents from two source indices to one destination index:
+
+```json
+POST _reindex
+{
+   "source":{
+      "index":[
+         "source_1",
+         "source_2"
+      ]
+   },
+   "dest":{
+      "index":"destination"
+   }
+}
+```
+
+## Reindex missing documents
+
+You can copy only documents missing from a destination index by setting the `op_type` option to `create`. To ignore all version conflicts of documents, set the `conflicts` option to `proceed`.
+
+```json
+POST _reindex
+{
+   "conflicts":"proceed",
+   "source":{
+      "index":"source"
+   },
+   "dest":{
+      "index":"destination",
+      "op_type":"create"
+   }
+}
+```
+
+## Reindex sorted documents
+
+You can copy certain documents after sorting specific fields in the document.
+
+This command copies the last 10 documents based on the `timestamp` field:
+
+```json
+POST _reindex
+{
+   "size":10,
+   "source":{
+      "index":"source",
+      "sort":{
+         "timestamp":"desc"
+      }
+   },
+   "dest":{
+      "index":"destination"
+   }
+}
+```
+
+## Transform documents during reindexing
+
+You can use an ingesting pipeline during the `reindex` operation to transform your data in flight as part of an ingest node pipeline process. The documents from the source index go through a pipeline that transforms them in some way before they get copied to the destination index.
```

Comment:
I think it's just "ingest pipeline." Also, this paragraph is pretty redundant. I'd cut off the final part of the sentence and say something like, "You can specify an ingest pipeline to transform your data during the `_reindex` operation." I'm not even sure we need the second sentence.